utils/swift_build_support/swift_build_support/products/swiftdriver.py

```diff
@@ -134,10 +136,27 @@ def run_build_script_helper(action, host_target, product, args):
         ]
     # Pass Cross compile host info
     if swiftpm.SwiftPM.has_cross_compile_hosts(args):
-        helper_cmd += ['--cross-compile-hosts']
-        for cross_compile_host in args.cross_compile_hosts:
-            helper_cmd += [cross_compile_host]
+        if targets.StdlibDeploymentTarget.get_target_for_name(
```

Comment:
Are you sure it's worth adding? All it would allow me to shave off is that one word `platform`, so it goes from `targets.StdlibDeploymentTarget.get_target_for_name(host_target).platform.is_darwin` to `targets.StdlibDeploymentTarget.get_target_for_name(host_target).is_darwin` or something.

Reply:
I meant a freestanding function, like `targets.is_darwin(host_target)`

Reply:
Oh, a free function that takes a string, will do.

Reply:
Instead, I noticed that [Gottesman added an `is_darwin_host()` method a couple months after I submitted this pull](https://github.com/apple/swift/pull/38124/files#diff-b84b1e74590d3a1234977cf25386482b020e325198c52f47625938c940a49e89R210), so I updated the first commit to use that instead.